apps/extension/webpack/webpack.common.js

```diff
@@ -70,13 +74,40 @@ export default {
     },
   },
   plugins: [
+    new webpack.CleanPlugin(),
     new webpack.ProvidePlugin({
       // Required by the `bip39` library
       Buffer: ['buffer', 'Buffer'],
     }),
+    new webpack.IgnorePlugin({
+      // Not required by the `bip39` library, but very nice
+      checkResource(resource) {
+        return /.*\/wordlists\/(?!english).*\.json/.test(resource);
+      },
+    }),
     new CopyPlugin({
-      patterns: [{ from: '.', to: '../', context: 'public' }],
-      options: {},
+      patterns: ['public', { from: 'bin', to: 'bin', transform: { cache: true } }],
```

Comment:
What's the thoughts on downloading to `/bin` + copying to `/dist` versus just downloading to `/dist`? Does it help us?

Reply:
makes it easy to delete dist without redownloading all keys. in general it would be nice to consider dist to be territory of the bundler, and avoid manual steps